### PR TITLE
Add GetTransactions

### PR DESCRIPTION
**Pull request by motxx**
See this PR comment: https://github.com/hyperledger/iroha/pull/754#issuecomment-351379734

### CODE REVIEW

File: source/includes/_queries.md [hunk 1]
```diff
@@ -287,6 +288,58 @@ Field | Description | Constraint
 -------------- | -------------- | --------------
 Transactions | an array of transactions for given account and asset | Committed transactions
 
+## Get transactions
+
+### Purpose
+
+When an information about transaction per its transaction's hash is needed, `GetTransactions` query is sent.
```
Comment (l4l) [inline source/includes/_queries.md]: Sounds too complicated. Maybe rephrase to (or smth similar):
`GetTransactions serves for retrieving information about transactions by its hashes`

File: source/includes/_queries.md [hunk 2]
```diff
@@ -287,6 +288,58 @@ Field | Description | Constraint
...
+    "creator_account_id": "admin@test",
+    "query_counter": 1,
+    "query_type" : "GetTransactions",
+    "tx_hashes": [string(64),…]
+}
+```
+
+Field | Description | Constraint
+-------------- | -------------- | --------------
+Transactions hashes | transactions' hashes to request transactions from  | (proto) array of 32 bytes, (json) 64 size of hex strings
```
Comment (l4l) [inline source/includes/_queries.md:327]: Imo we should follow in the field the exact name of the proto's message, ping @neewy 
Though out of this pr scope

**Comment by neewy** [comment]
Thank you for your contribution. I had to merge it in my code rather than merge it into docs branch. In future, there will be a contribution guide available.

Happy holidays! 🎄🎅 🎁 🕎
